aten/src/ATen/native/native_functions.yaml

```diff
@@ -2158,6 +2158,11 @@
   dispatch:
     MkldnnCPU: mkldnn_max_pool3d
 
+- func: quantized_max_pool1d(Tensor self, int[1] kernel_size, int[1] stride=[], int[1] padding=0, int[1] dilation=1, bool ceil_mode=False) -> Tensor
```

Comment:
just curious, can we dispatch from `max_pool1d` directly?

Reply:
I will factor out both 1d and 2d in later PRs if you don't mind -- it is giving me some errors when I do it now, need to look deeper